tox.ini

```diff
@@ -10,5 +10,5 @@ setenv =
 deps =
     .[all]
 commands =
-    py.test --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
+    py.test {posargs} --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
```

Comment:
I agree in principle, but there's _some_ reason to use `[]` instead of `{posargs}` that I can't remember at all. I'm trying to find it.